### DISCUSSION TITLE
开源之道（2016.5.31) - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#105] [Announcements]
# 开源之道（2016.5.31) - 「开源之道」

https://opensourceway.community/posts/opensource/daily_reading/2016/2016_5/2016_5_31/

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

能杀死的只有开源, 自由软件就没办法杀死了...
